app/views/layouts/modals/_new_message.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="modal new-message-modal" tabindex="-1" role="dialog" aria-hidden="true" style="display:block">
+  <div class="modal-dialog">
+    <%= form_for(Message.new, html: {role: 'form'}) do |f| %>
+        <div class="modal-content">
+          <div class="mu-modal-header" onclick="MessagesService.collapseNewMessageModal()">
```

Comment:
Ok, you are right, dismiss my comment. https://stackoverflow.com/questions/23217/whats-the-purpose-if-any-of-javascript-in-event-handler-tags